doc/book.toml

```diff
@@ -3,6 +3,7 @@ title = "Stylix"
 language = "en"
 
 [preprocessor.alerts]
+renderers = ["linkcheck"]
```

Comment:
As hinted at in https://github.com/Michael-F-Bryan/mdbook-linkcheck/issues/93, the admonitions are not properly rendered:

![image](https://github.com/user-attachments/assets/a25431ab-81dd-475b-b01f-fa74997b2586)

Did your workaround from https://github.com/Michael-F-Bryan/mdbook-linkcheck/issues/93#issuecomment-2961898171 properly work with the explicit `"html"` renderer, or does it simply silence the error while keeping the admonitions broken?

Reply:
The html renderer was missing from that list. Fixed. They seem to render.